package.json

```diff
@@ -18,13 +18,13 @@
     "postBuild": "node ./scripts/postBuild",
     "clean": "rimraf dist",
     "dev": "yarn storybook",
-    "format": "prettier --write \"src/**/*.{ts,tsx,json,js,jsx,md}\"",
-    "format:check": "prettier --list-different \"src/**/*.{ts,tsx,json,js,jsx}\"",
+    "format": "prettier --write \"src/**/*.{ts,tsx,json,js,jsx,md,mdx}\"",
+    "format:check": "prettier --list-different \"src/**/*.{ts,tsx,json,js,jsx,md,mdx}\"",
     "lint": "yarn lint:script && yarn lint:style",
-    "lint:script": "eslint --ext js --ext jsx --ext ts --ext tsx src",
+    "lint:script": "eslint --ext js --ext jsx --ext ts --ext tsx --ext mdx src",
     "lint:style": "stylelint src",
-    "build-storybook": "build-storybook -s public",
-    "storybook": "start-storybook -p 9001 -s ./src/assets -c .storybook",
+    "build-storybook": "build-storybook",
+    "storybook": "start-storybook -p 6006",
```

Comment:
https://storybook.js.org/docs/react/api/cli-options
The -s flag can be used to give static files in storybooks like images....this should solve a problem you faced previously